src/components/lists/RolesList/RolesList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Box from 'components/Box';
+import { Role, RoleData } from 'modules/roles/types';
+import { useCollectionData } from 'modules/data';
+import userDataList from 'mocks/userDataList';
+import RoleView from 'components/views/RoleView';
+import { ROLES } from 'utils/collectionNames';
+
+const RolesList = () => {
+  const { data } = useCollectionData<Role, RoleData>(ROLES);
+  return (
+    <Box style={{ margin: 20 }}>
```

Comment:
```suggestion
    <Box m={3}>
```

This will give a margin of 16 based on https://styled-system.com/. We using a `8px` grid bro